assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,192 @@
+  }
+
+  button:focus:not(:focus-visible) {
+    outline: 0;
+  }
+
+  .control-toggle {
+    align-self: flex-start;
+    background: transparent;
+    border: none;
+    color: var(--site-footer--color, $white);
+    cursor: pointer;
+    text-align: start;
+    white-space: nowrap;
+    width: 100%;
+
+    &::before,
+    &::after {
+      background-color: var(--site-footer--color, $white);
+      content: "";
+      display: inline-block;
+      width: 16px;
+    }
+
+    &::before {
+      height: 20px;
+      margin-inline-end: 8px;
+      mask: url("../../images/gp-g-logo.png") 0 0/16px 20px no-repeat;
+      vertical-align: middle;
+    }
+
+    &::after {
+      height: 12px;
+      margin-inline-start: 8px;
+      mask: url("../../images/chevron.svg") 0 0/16px 12px no-repeat;
+      transform: rotate(0.25turn);
+      transition: transform 300ms cubic-bezier(.86, 0, .07, 1);
+    }
+  }
+
+  .countries-list {
+    _-- {
+      background: var(--site-footer--background, $active-blue);
+    }
+    font-weight: 500;
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.5s cubic-bezier(0, 1.04, .38, .37) 0s;
+  }
+
+  .countries-list .container {
+    padding: 24px 24px $space-lg 16px;
+
+    @include medium-and-up {
+      padding: 24px 0 $space-lg;
+    }
+  }
+
+  &.open {
+    .country-selector-toggle-container {
+      border-bottom: 1px solid transparentize($white, 1);
+      transition-delay: 0s;
+    }
+
+    .control-toggle::after {
+      transform: rotate(-0.25turn);
+    }
+
+    .countries-list {
+      max-height: 5000px;
+      transition: max-height 1s linear 0s;
+    }
+  }
+
+  ul.countries {
+    column-count: 1;
+    list-style: none;
+    margin: 0;
+    padding-inline-start: $space-lg;
+
+    @include medium-and-up {
+      column-count: 3;
+    }
+
+    & li {
+      font-size: $font-size-sm;
+      line-height: 1;
+    }
+  }
+
+  li.international {
+    margin: 0;
+    padding: 0 0 $space-lg 0;
+  }
+
+  ul.lang-list {
+    font-weight: 400;
+    line-height: 24px;
+    list-style: none;
+    padding: $space-xxs 0 0 0;
+
+    @include medium-and-up {
+      margin-inline-end: $space-lg;
+    }
+  }
+
+  ul.lang-list > li {
+    display: inline-block;
+    font-size: $font-size-xxs;
+    line-height: 24px;
+  }
+
+  .control-close {
+    --country-selector--cross-- {
+      background-color: var(--site-footer--color, $white);
+    }
+
+    background-repeat: no-repeat;
+    border: none;
+    display: block;
+    float: right;
+    height: 16px;
+    mask: url("../../images/cross.svg") 0 0/16px 16px;
+    width: 16px;
+
+    html[dir="rtl"] & {
+      float: left;
+    }
+  }
+}
+
+.country-selector-toggle-container {
+  border-bottom: 1px solid transparentize($white, 0.5);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: 24px;
+  transition: border 0s linear 0.5s;
+  vertical-align: middle;
+  width: 100%;
+}
+
+.country-group-letter {
```

Comment:
This name seems like it refers to the letter for the group, but it refers to the group itself (which is grouped by letter). It's probably less confusing to call it `country-group`. Then you can auto generate the name of the `initial` class below and it will actually make sense `--country-group--initial`.